package.json

```diff
@@ -42,9 +42,6 @@
     "screwdriver-build-bookend": "^2.3.3"
   },
   "release": {
```

Comment:
We need `branches` under `release` property for semantic release.
https://github.com/screwdriver-cd/scm-github/blob/92590fd7fd66b7a0d5157b6ef44c80a475e02dca/package.json#L57-L59

Reply:
From the description of "semantic-release", it seems that the master branch is selected by default without setting the branches.
Should I explicitly set master to avoid unintended releases?

https://github.com/semantic-release/semantic-release/blob/master/docs/usage/configuration.md#options

Reply:
Currently, GitHub uses `main` branch as a default branch name. This might change the default behavior of semantic releases.
I think it is better to set explicitly branch name.

Reply:
I see.
I'll fix it.

Reply:
I fixed it.